### DISCUSSION TITLE
I don't understand the limitations of the Payouts system

### DISCUSSION THREAD

**Discussion by tyunderwood** [juspay/hyperswitch#5234] [Q&A]
Hello, 

I am interested in converting my small platform from Stripe Connect to using Hyperswitch. Obviously, we use Stripe Connect for payouts to our sub-merchants. It seems like Hyperswitch has a robust ability to replace this, but does this only work with Adyen? Our organization has already been rejected from using Adyen for being too small. It looks like Wise is also an option, but then on the info page Wise payouts show as 'not available', but then say they are supported elsewhere in the docs. Can someone illuminate for me how this works and what payout options aside from Adyen actually exist and are functional? Thanks.

**Comment by gorakhnathy7** [comment]
https://hyperswitch.io/pm-list
Kindly refer to the Payout processor section

**Reply by tyunderwood** [reply]
Hi there, yes I understand that's what I was looking at. The payout processor section says that every single payout processor is unavailable except for Adyen, and Adyen is impossible to qualify for unless you are a very large organization. So this means that there is no payout functionality except for Adyen, correct?

**Comment by kashif-m** [comment]
Hi @tyunderwood - we do have Stripe Connect integration for processing ACH and card payouts. Please note that this only works only for Custom connected accounts [[ref](https://docs.stripe.com/connect/custom-accounts)].

It's not feasible to process payouts using HyperSwitch when Stripe holds the raw payout data (in case of Standard and Express accounts).

What's not available in Stripe right now are webhook integrations. This will be added by end of this quarter (September end).
